EventListener/MimeTypeListener.php

```diff
@@ -50,7 +46,6 @@ public function onKernelRequest(GetResponseEvent $event)
         if (HttpKernelInterface::MASTER_REQUEST === $event->getRequestType()) {
             foreach ($this->mimeTypes['formats'] as $format => $mimeType) {
                 $request->setFormat($format, $mimeType);
-                $this->formatNegotiator->registerFormat($format, (array) $mimeType, true);
```

Comment:
This is no longer necessary because I now use the ``Request`` to find the format corresponding to a mime type.